### PR TITLE
[TASK] Require TYPO3 v12.4

### PR DESCRIPTION
**Pull request by brotkrueml**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,12 +12,12 @@
         }
     ],
     "require": {
-        "typo3/cms-backend": "^12.2 || dev-main",
```
Comment (linawolf) [inline composer.json:15]: I would prefere to require "^12.2 || dev-main" as we use this extension on the main branch of the documentation which always uses the latest main branch

Comment (brotkrueml) [reply composer.json:15]: Nope, when we tackled the last issues we should release a stable version. Which should only be compatible with v12.4, not with the upcoming v13. When we branch then we can add dev-main again (then without 12.4 compatibility):

Comment (linawolf) [reply composer.json:15]: The current main branch is not aiming to develop TYPO3 13, it has bugfixes of TYPO3 v12.4. The Core did not yet branch of v12 to 13

Comment (brotkrueml) [reply composer.json:15]: Yup, but there will be no new features for 12.4, so 12.4 is sufficient.

Comment (linawolf) [reply composer.json:15]: Bugfixes in the code, even those of comments in the code can and do positively influence the code-blocks we are printing. therefore I would always use dev-main for the main documentation branches

Comment (brotkrueml) [reply composer.json:15]: Okay, no problem, will close the PR for now.

Comment (linawolf) [reply composer.json:15]: Shouldn't we require "^12.4 || dev-main"? then?

Comment (brotkrueml) [reply composer.json:15]: Open a new PR. I do not agree. So the closing.

**Pull request closed without merging**